### PR TITLE
Add GANON_CLASSIFY

### LINKED ISSUE #3328: new module: GANON/CLASSIFY
### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by jfy133**
## PR checklist

Closes #3328 

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/ganon/classify/main.nf
Comment (sofstam) [inline modules/nf-core/ganon/classify/main.nf:36]:
```suggestion
        --output-prefix ${prefix} \\
```
I am thinking about consistency in all arguments

File: modules/nf-core/ganon/classify/main.nf
Comment (jfy133) [reply modules/nf-core/ganon/classify/main.nf:12]: Actually good point, no probably not. Can use multiMap

**Review by sofstam — APPROVED** [review]
Lgtm once the tests pass and merge master into this branch :)

**Pull request merged**
